packages/orbit-components/src/Slider/utils/calculateCountOf.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Value, Data } from "..";
+
+// eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+declare const CalculateCountOf: (data: Data, value: Value, min: number) => [number, number];
+
+export default CalculateCountOf;
```

Comment:
FYI: the Flow declaration also exports the `CalculateCountOf` as a named export, but that's pointless because you can always get it with `typeof`, so I don't think it's worth mirroring that here.